src/Symfony/Component/Security/Core/Resources/translations/security.lt.xlf

```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Netinkama arba pasibaigusio galiojimo laiko prisijungimo nuoroda.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Per daug nepavykusių prisijungimo bandymų, pabandykite dar kartą po %minutes% minutės.</target>
```

Comment:
If it make sense, You can write "one minute" here instead. 

Ie (google translate) "vieną minutę".

Reply:
> If it make sense, You can write "one minute" here instead.
> 
> Ie (google translate) "vieną minutę".

actually "po 1 minutės" :)

Reply:
If you prefer "1 minute" over "one minute" then I'm happy with this PR as it is. 

=)